src/test/java/org/cyclonedx/parsers/JsonParserTest.java

```diff
@@ -70,12 +70,12 @@ public void testParsedObjects12Bom() throws Exception {
         Assert.assertEquals(0, bom.getMetadata().getComponent().getSwid().getTagVersion());
         Assert.assertEquals(false, bom.getMetadata().getComponent().getSwid().isPatch());
         Assert.assertEquals("Acme, Inc.", bom.getMetadata().getManufacture().getName());
-        Assert.assertEquals("[\"https://example.com\"]", bom.getMetadata().getManufacture().getUrl()); // TODO: fix this
+        Assert.assertEquals("https://example.com", bom.getMetadata().getManufacture().getUrls().get(0)); // TODO: fix this
```

Comment:
what needs to be fixed here?

Reply:
I don't think anything other than removing the comment, lol. @stevespringett this was a fix for urls actually being an array, not just a single URL.

Reply:
351cb65